src/promptflow/promptflow/_sdk/_orm/otel.py

```diff
@@ -0,0 +1,38 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from sqlalchemy import TEXT, Column, Index
+from sqlalchemy.orm import declarative_base
+
+from promptflow._sdk._constants import SPAN_TABLENAME
+
+from .retry import sqlite_retry
+from .session import otel_mgmt_db_session
+
+Base = declarative_base()
+
+
+class Span(Base):
+    __tablename__ = SPAN_TABLENAME
+
+    span_id = Column(TEXT, primary_key=True)
```

Comment:
according to [OTLP spec](https://opentelemetry.io/docs/specs/otlp/#duplicate-data), shall we remove this primary key constraint? or make this a known ignorable exception?